src/schema-templates/recipe.schema.php

```diff
@@ -11,7 +11,7 @@
 	"author": {
 		"@id": "%%author_id%%"
 	},
-	"description": {{inner-blocks-html blocks={ "core/paragraph": "content" } null-when-empty=true allowed-tags=[ "h1","h2","h3","h4","h5","h6","br","a","p","b","strong","i","em" ] }},
+	"description": {{inner-blocks-html blocks={ "yoast/recipe-description": "description" } null-when-empty=true allowed-tags=[ "h1","h2","h3","h4","h5","h6","br","a","p","b","strong","i","em" ] }},
```

Comment:
I'm not sure about these allowed tags. Are they necessary?
(I know the `description` for Jobs has them as well, but that's not enough motivation)

Reply:
Decided with @Xyfi that he will make a new ticket to investigate this for the recipe description and jobs description.